step.yml

```diff
@@ -53,6 +53,8 @@ inputs:
       #!/usr/bin/env bash
       # fail if any commands fails
       set -e
+      # make pipelines' return status equal the last command to exit with a non-zero status
```

Comment:
Is this always non-zero status 🤔 ? Shouldn't we review this comment with TW?

Reply:
I phrased this using a documentation I found on `pipefail`:
>If `pipefail` is enabled, the pipeline’s return status is the value of the last (rightmost) command to exit with a non-zero status, or zero if all commands exit successfully. The shell waits for all commands in the pipeline to terminate before returning a value.

We can add a `(0 otherwise)` part at the end if you think it makes this explicit.

Reply:
Added it, please check!